tests/Feature/CanonicalUrlTest.php

```diff
@@ -18,61 +19,81 @@ function inProduction()
 afterEach(fn () => App::detectEnvironment(fn () => 'testing'));
 
 test('pages without a canonical url explicitly set fall back to the current url', function () {
+    $expectedHtml = new HtmlString('<link rel="canonical" href="http://localhost/register" />');
```

Comment:
@joedixon  @driesvints 
AssertSee full link without browserkit are causing some errors in the quotes, so I've updated to expectedHtml using the HtmlString. 

Not sure if is the better fix, tell me if is not.

Reply:
Looks good!